### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by layomia**

### CODE REVIEW

File: .github/CODEOWNERS
Comment (eerhardt) [inline .github/CODEOWNERS:6]: Is @ahsonkhan still valid here?

Comment (danmosemsft) [reply .github/CODEOWNERS:6]: That presumably is up to @ahsonkhan, since all it does is set him as a reviewer. Any reviews he does are greatly appreciated, but not expected...

**Review by eiriktsarpalis — APPROVED** [review]

File: .github/CODEOWNERS
Comment (ahsonkhan) [reply .github/CODEOWNERS:6]: I am happy to get tagged for reviews and help from time to time, which is why I left the code owners as is :)

If there is a pressing reason to update the list, that would be OK with me as well.

Comment (layomia) [reply .github/CODEOWNERS:6]: Thanks. Will leave @ahsonkhan as a code owner - we can update in the future if needed.

Comment (eerhardt) [reply .github/CODEOWNERS:6]: Thanks @ahsonkhan!

**Pull request merged**
